src/utils/formUtils.js

```diff
@@ -113,3 +113,21 @@ export const getEntityTags = props => {
 
   return get(entity, ['tags', 'tagList'], []);
 };
+
+/**
+ * Formats Decorator Value according requirements
+ *
+ * @param {string} currentValue
+ * @param {string} newValue
+ * @param {boolean} isNeedToWrapInQuotes
+ * @return {string} Decorated input value
+ */
+export const formatDecoratorValue = (currentValue, newValue, isNeedToWrapInQuotes) => {
+  const decoratorValueRegExp = /"(\\.|[^"\\])*"/g;
```

Comment:
Could you please clarify what string you expect to match with this regexp?
`/"[^"]+"/g` - maybe it can be replaced with such regexp?

Reply:
done